generator/lib/builder/om/PHP5ObjectBuilder.php

```diff
@@ -8,7 +8,7 @@
  * @license    MIT License
  */
 
-require_once dirname(__FILE__) . '/ObjectBuilder.php';
+require_once __DIR__ . '/ObjectBuilder.php';
```

Comment:
`__DIR__` is set for PHP 5.3 and up (http://php.net/manual/en/language.constants.predefined.php), current Propel requirement is 5.2.4 and up (https://github.com/propelorm/Propel/blob/master/composer.json#L17) so this won't work for all users (sandly, because I like the change :)  )